### DISCUSSION TITLE
Support for notetrack "animation" keys (3ds format, primarily)

### DISCUSSION THREAD

**Discussion by keelanstuart** [assimp/assimp#5380] [Ideas]
Is there any interest in supporting a "note track" for embedding timed commands in animations? 3ds max lets you create this kind of track and will write it for certain formats... but I do not know if other modelling packages have a similar feature. If you've a game dev and have never used notes, they can change your life by empowering your artists to spawn effects or play sounds at a certain point in the animation, etc.

I guess what I want to know is, @kimkulling would you accept a PR that added this? Maybe under a feature flag? Obviously, I can do it on my fork... I'm just curious.

Thanks!

**Comment by kimkulling** [comment]
Hi, 
honestly speaking I do not have a clue how this kind of animation works. Sou getting a PR to discuss this would be great.
Kim

**Reply by keelanstuart** [reply]
Hi again-
Instead of having a *spatial transform* attached to a node at a given time/frame, a string is attached there instead... you can use it for anything, but I found it perfect for running scripts at a particular time. When I make the changes, I will submit a PR.
Thanks!
-Keelan

EDIT: I may have to abort - I've always used the feature from a 3dsmax plug-in. After using the standard export for a model with a note track and some basic animation into 3ds, fbx, and dae formats -- at least one of which I would have expected to see the data in -- it was not there. I guess this means that it isn't viable for assimp. Sadness.
Anyway, cheers and thanks for responding.
